data/virtualization/autoyast/guest_12.xml.ep

```diff
@@ -327,6 +327,10 @@
 mkdir -p -m 700 /root/.ssh
 echo 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC0XMR9r8NNr58+8BmbD+CyTQd9iQqiIYjY45gViEql0+R5rahWHQBxuSAxkJbPMLudym1ffdS3+ODuBVO8nHABZ7eGaPZJQQTO0Y2bpJXCis/0GxPUD94EJ+NI/4/qzpyuw5CbfO2MazXeF38mBRO/ceZ54ZIkZMuqTHq5kp+rhq2KyUBxum9tWC7uHh0JFCoRro28uRSgNn3uVSMe1pAihdqwlZ6CaDwfN/3+aEqDTz/e/E41AKJjDx7DaCO6alKbkn2V8D2althjnB7ZtKNsYfkiQ6V9qu9lEtgeFQeEZJklWPbq+CYLcvr7Fys8M56AANqBza5Kc8a7ALZZNEbrbbmsCy+avS/Ptft7LXNjDfNTUXQ+NRDNHtvc9UmHOx9cdokgU54hxAfxv8K2scvaWeBQJHAu76oNMrnGxKIMdNBH0ZbNudSgLlHV9Z/PIYYXN7XxHcJuIg9hjo2ryUdPdRA4KGaeZN9rXnr088qJUtDJj0MBaRSCZtq5zWaJyUk=' >> /root/.ssh/authorized_keys
 chmod 600 /root/.ssh/authorized_key
+touch /etc/init.d/after.local && chmod +x /etc/init.d/after.local
+echo "#!/bin/bash
+ip -br a | grep eth0 | awk '{print /$3}' | cut -d'/' -f1  > /root/$(hostname)
+curl -k -u root:{{PASS}} -T /root/$(hostname) sftp://192.168.122.1/root/guests_ip/ --ftp-create-dirs" > /etc/init.d/after.local
```

Comment:
It's good to replace "192.168.122.1" with host sut_ip

Reply:
Do we have a variable containing the host ip somewhere?